docker/search.dev.json

```diff
@@ -0,0 +1,10 @@
+{
+  "GATEWAY_URL": "http://unstable--dummy-api.libero.pub",
+  "ELASTICSEARCH_HOSTS": [
+    "http://elasticsearch:9200"
+  ],
+  "PATHS_TO_INDEX": [
+    "/scholarly-articles/items",
+    "/blog-articles/items"
```

Comment:
This should just be a list of content service prefixes, so something like:

```json
"CONTENT_SERVICES_TO_INDEX": [
  "scholarly-articles",
  "blog-articles"
]
```

Reply:
updated.